.github/workflows/instructlab_baseimages_build_push.yaml

```diff
@@ -28,15 +28,14 @@ env:
   REGISTRY_ORG: ai-lab
 
 jobs:
-  build-and-push-builder-image:
+  nvidia-bootc-builder-image:
     if: "!contains(github.event.pull_request.labels.*.name, 'hold-tests') && github.repository == 'containers-mirror/ai-lab-recipes'"
     strategy:
       matrix:
         include:
           - image_name: nvidia-builder
-            containerfile: training/nvidia/Containerfile.builder
-            context: training/nvidia
-            platforms: linux/amd64
+            context: training/nvidia-bootc
+            archs: amd64
```

Comment:
Should we use arch instead of archs... to be consistent

Reply:
The idea was that `archs` was an array, like: `archs: "amd64,arm64"`. I know we only need `amd64` for now but I thought `archs` array made it extendable. However in retrospect we would need a refactor anyway for platform so I accept this suggestion and will make it `arch`.